extensions/wikia/Wall/Wall.i18n.php

```diff
@@ -107,6 +107,8 @@
 	'wall-action-monobook-login' => 'Please log in to post a message on this wiki',
 	'wall-posting-message-failed-title' => 'An error occurred while posting a message',
 	'wall-posting-message-failed-body' => 'Copy your work and try again later',
+	'wall-posting-message-failed-filter-title' => 'Inapropriate content detected', //TODO: set proper message
```

Comment:
What should be the message (and the one below)? Or who should be pinged about this?